frontend/public/components/about-modal.tsx

```diff
@@ -87,7 +88,17 @@ const AboutModalItems: React.FC<AboutModalItemsProps> = ({ closeAboutModal }) =>
             {window.SERVER_FLAGS.kubeAPIServerURL}
           </TextListItem>
 
-          <ServiceLevel clusterID={clusterID}>
+          <ServiceLevel
+            clusterID={clusterID}
+            loading={
+              <>
+                <TextListItem component="dt" />
```

Comment:
While this satisfies the a11y check technically, it sort of bypasses the a11y check in spirit since the `<dt>` and `<dd>` are empty.  @kdoberst, WDYT?

Reply:
Thinking about this further, perhaps
```
<>
  <TextListItem component="dt">{useServiceLevelTitle()}</TextListItem>
  <TextListItem component="dd">
    <Skeleton screenreaderText={t('public~Loading')} />
  </TextListItem>
</>
```
is one way to address?

Reply:
This approach has the added advantage of making the render of this item consistent with the other items in the cluster overview details card

https://user-images.githubusercontent.com/895728/178515451-727024d0-ffb5-44b7-a48d-a6a4851858a5.mov

Reply:
I agree it's not "perfect", but it is better than before. I updated the screenshots in my first comment, you can take a look.

Reply:
Your solution renders a loading indicator before we know that we want to show an SLA headline, or? So this is different than before when we don't show a title before we know that something is rendered.

It's not wrong, but it's a change, I tried here not to change the implementation (for the user).

Reply:
Since I went down the path of coding this up, I put up a PR against to include the fix should we want to go this route.  https://github.com/jerolimov/console/pull/6